sim/firesim-lib/src/main/scala/passes/ILATopWiring.scala

```diff
@@ -19,7 +19,7 @@ import firrtl.transforms.TopWiring._
     This also has an option to pass a function as a parmeter to generate custom output files as a result of the additional ports
   * @note This *does* work for deduped modules
   */
-class ILATopWiringTransform(dir: File = new File("/tmp/")) extends Transform {
+class ILATopWiringTransform(dir: File = new File("/tmp/"), depth: Int = 1024) extends Transform {
```

Comment:
Another nit: reading this, depth seems fairly generic. Naming it ila_window_depth might be more explicit?

Reply:
I can name it data_depth, to match what the actual variable is called

Reply:
sure